specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,190 @@
+language. It must be a single language tag; a delimited list of language tags will result in default
+language being used.
+
+The PrimaryLanguageOverride property should only be set to languages available for the app.
+
+When your app gets the value, PrimaryLanguageOverride returns either a single language tag (if your
+app has previously set the property) or an empty string.
+
+# Remarks
+
+Apps normally run with language settings determined by the system by comparing the languages
+supported by the app with the language preferences of the user. The PrimaryLanguageOverride property
+is used to override that behavior by setting a single specific language.
+
+For packaged apps, `PrimaryLanguageOverride` setting is a wrapper for
+[Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride)
+property and behaves the same. It is persisted between app sessions.
+
+For un-packaged apps, `PrimaryLanguageOverride` setting is not persisted between sessions. It should
+be set each time the app is loaded. You should set PrimaryLanguageOverride setting in early stage of
+app loading, before any resource is loaded.
+
+The property should only be set based on user input presented in settings UI. It can be read at any
+time. If the property has never been set, it returns an empty string.
+
+When you set the PrimaryLanguageOverride, this will be immediately reflected in resources loaded
+afterwards, both resources loaded from the code and the resources loaded by XAML. However, this
+change may not take effect immediately on resources already loaded in the app UI. To make sure the
+app responds to such changes, actions may be needed to reload resources. Those requirements may vary
+depending on the UI framework used by the app, and it may be necessary to restart the app.
+
+Example:
+
+```c#
+public partial class App : Application
+{
+    // ...
+    public App()
+    {
+        // Set the primary language override to German (Germany)
+        // This will cause the app to use the German (Germany) resources
+        // Both resources loaded manually with ResourceLoader and XAML resources fetched with x:Uid
+        // will be German (Germany)
+        Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages.PrimaryLanguageOverride = "de-DE";
+
+        Microsoft.Windows.ApplicationModel.Resources.ResourceLoader resourceLoader = new Microsoft.Windows.ApplicationModel.Resources.ResourceLoader();
+        var resourceString = resourceLoader.GetString("SampleString");
+
+        this.InitializeComponent();
+    }
+    // ...
+}
+```
+
+# API Details
+
+```c# (but really MIDL3)
+namespace Microsoft.Windows.ApplicationModel.Resources
+{
+  // ...
+  [contract(MrtCoreContract, 2)]
+  runtimeclass ApplicationLanguages
+  {
+      static IVectorView<String> Languages { get; };
+      static IVectorView<String> ManifestLanguages { get; };
```

Comment:
`IVector<String>`?

Reply:
To align with Windows.globalization property used in the background . and get IReadOnlyList in C#

Reply:
Keep as-is to match existing api.